Makefile

```diff
@@ -7,9 +7,12 @@ help: ## Displays help.
 	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n\nTargets:\n"} /^[a-z0-9A-Z_-]+:.*?##/ { printf "  \033[36m%-10s\033[0m %s\n", $$1, $$2 }' $(MAKEFILE_LIST)
 
 .PHONY: test
+test: ## Runs all Go unit tests.
+export GOCACHE=/tmp/cache
```

Comment:
Hm, isn't this file autogenerated by bingo?

Reply:
This target does not use bingo in any way. This is just to always run tests and not cache results. 🙂